ajax/massreception.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/*
+ LICENSE
+ This file is part of the order plugin.
+ Order plugin is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+ Order plugin is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ You should have received a copy of the GNU General Public License
+ along with GLPI; along with Order. If not, see <http://www.gnu.org/licenses/>.
+ --------------------------------------------------------------------------
+ @package   order
+ @author    the order plugin team
+ @copyright Copyright (c) 2010-2015 Order plugin team
+ @license   GPLv2+
+            http://www.gnu.org/licenses/gpl.txt
+ @link      https://forge.indepnet.net/projects/order
+ @link      http://www.glpi-project.org/
+ @since     2009
+ ---------------------------------------------------------------------- */
```

Comment:
Header are not up-to-date. Please use `vendor/bin/licence-headers-check --fix`.

Reply:
done